reconcile/utils/oc.py

```diff
@@ -591,8 +599,15 @@ def _run(self, cmd, **kwargs):
             if kwargs.get('apply'):
                 if 'Invalid value: 0x0' in err:
                     raise InvalidValueApplyError(f"[{self.server}]: {err}")
-                if 'Invalid value: ' in err and ': field is immutable' in err:
-                    raise FieldIsImmutableError(f"[{self.server}]: {err}")
+                if 'Invalid value: ' in err:
```

Comment:
What happens if this codebranch is taken but none of the child-ifs get executed? should this if have a corresponding else?

Reply:
it goes through the `StatusCodeError` condition

Reply:
i see!
lgtm then :)